src/app/models/evt-models.ts

```diff
@@ -849,3 +849,129 @@ export class SourceDesc extends GenericElement {
 }
 
 export class Extent extends GenericElement { }
+
+export class EncodingDesc extends GenericElement {
+    structuredData: boolean;
+    projectDesc: ProjectDesc[];
+    samplingDecl: SamplingDecl[];
+    editorialDecl: EditorialDecl[];
+    tagsDecl: TagsDecl[];
+    styleDefDecl: Array<ParseResult<GenericElement>>; // TODO: Add specific type when styleDefDecl is handled
+    refsDecl: RefsDecl[];
+    classDecl: Array<ParseResult<GenericElement>>; // TODO: Add specific type when classDecl is handled
+    geoDecl: Array<ParseResult<GenericElement>>; // TODO: Add specific type when geoDecl is handled
+    unitDecl: Array<ParseResult<GenericElement>>; // TODO: Add specific type when unitDecl is handled
+    schemaSpec: Array<ParseResult<GenericElement>>; // TODO: Add specific type when schemaSpec is handled
+    schemaRef: Array<ParseResult<GenericElement>>; // TODO: Add specific type when schemaRef is handled
+}
+
+export class ProjectDesc extends GenericElement {
+    content: Paragraph[];
+}
+
+export class SamplingDecl extends GenericElement {
```

Comment:
Use type alias for this

Reply:
If I use type alias (`export type SamplingDecl = ProjectDesc;`), then I get an error in the parser in the setting of `type` property:
```
parse(xml: XMLElement): SamplingDecl {
    return {
      ...super.parse(xml),
      type: SamplingDecl,
      content: queryAndParseElements<Paragraph>(xml, 'p'),
    };
  }
```
The error says: «'SamplingDecl' only refers to a type, but is being used as a value here.»